### PR TITLE
Fix edit inline mode for the website module

### PR DESCRIPTION
**Pull request by Daoud-mohamed**
# Fix edit inline mode for the website module
Fix edit inline mode for the website module

### CODE REVIEW

File: htdocs/core/ajax/editinline.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	if ($res) {
+		global $dolibarr_main_data_root;
+		$pathofwebsite = $dolibarr_main_data_root.($conf->entity > 1 ? '/'.$conf->entity : '').'/website/'.$website_ref;
+		$filetpl = $pathofwebsite.'/page'.$objectpage->id.'.tpl.php';
+
+		$result = dolSavePageContent($filetpl, $objectwebsite, $objectpage, 1);
+		if (!$result) {
+			print "Failed to write file " . $filetpl . ".";
+			$db->rollback();
+			exit;
```
Comment (eldy) [inline htdocs/core/ajax/editinline.php:83]: The exit will bypass the db->close at the end.

File: htdocs/core/ajax/editinline.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+if (!empty($action) && $action === 'updatedElementContent' && !empty($content) && !empty($element_id) && !empty($website_ref) && !empty($page_id)) {
+	$db->begin();
+
+	// Page object
+	$objectpage = new WebsitePage($db);
+	$res = $objectpage->fetch($page_id);
+	if (!$res) {
+		print "Cannot find page with ID = " . $page_id . ".";
+		exit;
```
Comment (eldy) [inline htdocs/core/ajax/editinline.php:60]: A db->begin was done, we can't make an exit before the db->rollback

File: htdocs/core/ajax/editinline.php [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+$action = GETPOST('action', 'alpha');
+$website_ref = GETPOST('website_ref');
+$page_id = GETPOST('page_id');
+$content = GETPOST('content', 'none');
+$element_id = GETPOST('element_id');
+$element_type = GETPOST('element_type');
+
+
+if (!empty($action) && $action === 'updatedElementContent' && !empty($content) && !empty($element_id) && !empty($website_ref) && !empty($page_id)) {
```
Comment (eldy) [inline htdocs/core/ajax/editinline.php:52]: Any action must add also a test on 
`... && $usercanmodify ...`

with before 
`$usercanmodify = $user->getRight('website', 'write');`

**Comment by Daoud-mohamed** [comment]
@eldy fixed !

**Pull request merged**